shared/checks/oval/accounts_passwd_has_interactive_users.xml

```diff
@@ -0,0 +1,64 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwd_has_interactive_users" version="1"> 
+    {{{ oval_metadata("Checks /etc/passwd for interactive users.") }}}
```

Comment:
I think this macro doesn't work when it is in `shared/checks/oval`.

You'll need to do it as done in other checks in `shared/checks/oval`: 
e.g.: https://github.com/ComplianceAsCode/content/blob/3d9ba049c998569d8f0bc74c14c76667fe6a04ea/shared/checks/oval/audit_rules_networkconfig_modification_domainname.xml
```
<metadata>
      <title>Record Events that Modify the System's Network Environment</title>
      <affected family="unix">
        <platform>multi_platform_all</platform>
      </affected>
      <description>The network environment should not be modified by anything other than
      administrator action. Any change to network parameters should be audited.</description>
</metadata>
```